crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -853,7 +853,9 @@ impl ExecContext {
 
         if self.experimental_ui {
             if let TaskOutput::UI(task) = output_client {
-                task.start();
+                // We should send over the output mode for the task here :)
+                let output_logs = self.task_cache.output_logs().into();
```

Comment:
I don't understand this comment. Is this a TODO or something that the next line is doing?

Reply:
Yeah that was a TODO I missed